src/vivarium_gates_iv_iron/data/loader.py

```diff
@@ -66,6 +66,7 @@ def get_data(lookup_key: str, location: str) -> pd.DataFrame:
         data_keys.POPULATION.TMRLE: load_theoretical_minimum_risk_life_expectancy,
         data_keys.POPULATION.ACMR: load_standard_data,
         data_keys.POPULATION.PLW_LOCATION_WEIGHTS: get_pregnant_lactating_women_location_weights,
+        data_keys.POPULATION.WRA_LOCATION_WEIGHTS: get_wra_location_weights,
```

Comment:
Anticipating a nitpick: Should we change this to `get_women_reproductive_age_location_weights` or the above to `get_plw_location_weights`?

Reply:
I think I prefer the verbose method name, but yeah consistency would be nice.

Reply:
Changing this.